schemas/2016-04-30-preview/Microsoft.Compute.json

```diff
@@ -0,0 +1,4781 @@
+        }
+      },
+      "description": "Describes a virtual machine scale set sku."
+    },
+    "SshConfiguration": {
+      "type": "object",
+      "properties": {
+        "publicKeys": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/SshPublicKey"
+              }
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The list of SSH public keys used to authenticate with linux based VMs."
+        }
+      },
+      "description": "SSH configuration for Linux based VMs running on Azure"
+    },
+    "SshPublicKey": {
+      "type": "object",
+      "properties": {
+        "path": {
+          "type": "string",
+          "description": "Specifies the full path on the created VM where SSH public key is stored. If the file already exists, the specified key is appended to the file."
+        },
+        "keyData": {
+          "type": "string",
+          "description": "Certificate public key used to authenticate to the VM through SSH. The certificate must be in Pem format with or without headers."
+        }
+      },
+      "description": "Contains information about SSH certificate public key and the path on the Linux VM where the public key is placed."
+    },
+    "StorageProfile": {
+      "type": "object",
+      "properties": {
+        "imageReference": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/ImageReference"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The image reference."
+        },
+        "osDisk": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/OSDisk"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The OS disk."
+        },
+        "dataDisks": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/DataDisk"
+              }
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The data disks."
+        }
+      },
+      "description": "Describes a storage profile."
+    },
+    "SubResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource Id"
+        }
+      }
+    },
+    "UpgradePolicy": {
+      "type": "object",
+      "properties": {
+        "mode": {
```

Comment:
"mode" is a required property in the previous version, and can be any string value. Can we confirm on the change?